### PR TITLE
Bump operator dependencies to 1.14.

### PR DESCRIPTION
**Pull request by mfojtik**

### CODE REVIEW

File: glide.yaml [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
 - package: k8s.io/kube-aggregator
-  version: kubernetes-1.13.4
+  version: kubernetes-1.14.0
 - package: k8s.io/code-generator
-  version: kubernetes-1.13.4
+  version: kubernetes-1.14.0
 - package: k8s.io/apiextensions-apiserver
   repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
-  version: origin-4.1-kubernetes-1.13.4 # pinned to get nullable patch
+  version: origin-4.1-kubernetes-1.14.0 # pinned to get nullable patch
```
Comment (deads2k) [inline glide.yaml]: Can't we switch to normal upstream now?

Comment (sttts) [reply glide.yaml]: we can

Comment (mfojtik) [reply glide.yaml]: fixed.

**Comment by mfojtik** [comment]
@sanchezl can you please check that operator e2e failure?

**Pull request merged**